app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,51 @@
+class FoodsController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @foods = Food.all
+  end
+
+  def new; end
+
+  def create
+    @food = Food.new(food_params)
+    @food.user_id = current_user.id
+
+    if @food.save
+      redirect_to foods_path, notice: 'Food was successfully created.'
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
```

Comment:
- [ ] Creating a new food entry doesn't work as expected and shows an error which I have attached below 👇 Please try to fix this bug because the ability to create food entries is essential to the complete testing of this application 🙏 
![image](https://user-images.githubusercontent.com/46839548/209442632-4427fe45-771f-4082-873f-36259f55e4dc.png)